playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -352,6 +363,7 @@
     _cmpnt_runcommand: "./tools/deployment/multinode/140-compute-kit.sh"
   environment:
     OSH_EXTRA_HELM_ARGS_NOVA: "{{ lookup('env', 'OSH_EXTRA_HELM_ARGS_NOVA') | default('', True) }} --values /tmp/socok8s-nova.yaml"
+    OSH_EXTRA_HELM_ARGS_NEUTRON: "{{ lookup('env', 'OSH_EXTRA_HELM_ARGS_NOVA') | default('', True) }} --values /tmp/socok8s-neutron.yaml"
```

Comment:
I think you might mean 'OSH_EXTRA_HELM_ARGS_NEUTRON'

Reply:
:+1: